components/organisms/Assessment/AssessmentFinished/AssessmentFinished.tsx

```diff
@@ -44,9 +45,7 @@ export const AssessmentFinished: FC<AssessmentFinishedProps> = ({
     const nextLesson = getNextLesson();
 
     if (nextLesson) {
-      router.push(
-        "/dashboard"
-      );
+      router.push("/dashboard");
     } else {
       console.log("No hay más lecciones disponibles");
```

Comment:
review

Reply:
@DanielCalleCO  changed for an alert that can give information to the user